Detections/SigninLogs/SigninBruteForce-AzurePortal.yaml

```diff
@@ -76,5 +75,5 @@ entityMappings:
     fieldMappings:
       - identifier: Address
         columnName: IPCustomEntity
-version: 1.0.1
+version: 1.0.2
```

Comment:
This is a pretty major change, let's go with 2.0.

Reply:
Thanks @shainw. Updated the version as suggested.